service/src/main/resources/schema/development/schema.json

```diff
@@ -138,6 +138,17 @@
         }
       }
     },
+    "generatedBy": {
+      "type": "object",
+      "properties": {
+        "TerraCore:hasAssayCategory": {
+          "type": "string"
+        },
+        "TerraCore:hasDataModality": {
+          "type": "string"
```

Comment:
@rsc3 can you keep us up to date on the development of the data modality ontology?

The goal is to have a dictionary of terms we can use to add to our schema to help data ingesters conform to a common model.